src/test/end-to-end/pages/amendCaseDetails/amendCaseDetails.js

```diff
@@ -1,13 +1,30 @@
 'use strict';
 const commonConfig = require('../../data/commonConfig.json');
+const caseDetails = require("../amendCaseDetails/helper/caseDetails");
+const testConfig = require("../../../config");
 
-module.exports = async function (clerkResponsible, physicalLocation, conciliationTrack) {
+module.exports = async function (clerkResponsible, physicalLocation, suggestedHearingVenue) {
 
     const I = this;
-    await I.selectOption('#clerkResponsible', clerkResponsible);
-    await I.selectOption('#fileLocation', physicalLocation)
-    await I.selectOption('#conciliationTrack', conciliationTrack)
-    await I.click(commonConfig.continue);
-    await I.click(commonConfig.continue);
-    await I.click(commonConfig.submit);
+    //navigate to the case details event and enter fields
+    I.waitForText('Case Details', testConfig.TestTimeToWaitForText);
+    I.waitForElement('#receiptDate-day', testConfig.TestTimeToWaitForText)
```

Comment:
Are we not checking for any other Labels on this page...

Reply:
For the case details event itself i'm not checking for any labels. I do the assertion (check for labels) on the case details tab instead. (src/test/end-to-end/pages/amendCaseDetails/helper/caseDetails.js)